.vscode/settings.json

```diff
@@ -30,5 +30,6 @@
     },
     "clangd.arguments": [
         "--header-insertion=never"
-    ]
+    ],
+    "cmake.configureOnOpen": false
```

Comment:
```suggestion
    ]
```

Reply:
change is outdated..cant commit this suggetion.